Cubical/Structures/Group/Base.agda

```diff
@@ -163,3 +179,66 @@ makeGroup-left {A = A} id comp inv set assoc lUnit lCancel =
           ≡⟨ lUnit a ⟩
         a
           ∎
+
+
+open Group
+open IsGroup
+
+open Group
+η-Group : {G H : Group {ℓ}}
+       → (p : ⟨ G ⟩ ≡ ⟨ H ⟩)
+       → (q : PathP (λ i → p i) (0g G) (0g H))
+       → (r : PathP (λ i → (p i) → (p i) → (p i)) (_+_ G) (_+_ H))
+       → (s : PathP (λ i → p i → p i) (-_ G) (-_ H))
+       → PathP (λ i → IsGroup (q i) (r i) (s i)) (isGroup G) (isGroup H)
+       → G ≡ H
+Carrier (η-Group p _ _ _ _ i) = p i
+0g (η-Group _ q _ _ _ i) = q i
+_+_ (η-Group _ _ r _ _ i) = r i
+- η-Group _ _ _ s t i = s i
+isGroup (η-Group _ _ _ _ t i) = t i
+
+isSetCarrier : ∀ {ℓ} → (G : Group {ℓ}) → isSet ⟨ G ⟩
+isSetCarrier G = IsSemigroup.is-set (IsMonoid.isSemigroup (isMonoid (isGroup G)))
+
+open import Cubical.Foundations.HLevels
```

Comment:
Move import to the top. Also, do you need to open both IsMonoid and IsSemigroup?

Reply:
Yeah, I need them for this to run. I could of course skip them, but the lines get so long and ugly if I do (they are already pretty ugly as they are).